assets/subworkflows/espresso/band_structure_average_esp.yml

```diff
@@ -35,10 +49,19 @@ units:
 -   config:
         execName: pp.x
         flavorName: pp_electrostatic_potential
-        name: ESP
+        name: Electrostatic Potential (ESP)
     type: executionBuilder
 -   config:
         execName: average.x
-        flavorName: average
+        flavorName: averaged_potential
         name: average ESP
+        flowchartId: average-electrostatic-potential
     type: executionBuilder
+-   config:
```

Comment:
maybe just `averaged_electrostatic_potential` as a name? band structure is extracted, but is not a focus

Reply:
or `averaged_electrostatic_potential_via_band_structure`